ios/private/cocoapods_publishing.bzl

```diff
@@ -9,32 +9,6 @@ _TEMPLATE_TRUNK_PUSH = Label("//ios/private:pod_trunk_push.sh")
 _TEMPLATE_SPM_PUBLISH = Label("//ios/private:unzip_and_tag.sh")
 _TEMPLATE_PUSH = Label("//ios/private:push.sh")
 
-def _spm_publish_impl(ctx):
-    output = ctx.actions.declare_file("publish.sh")
-
-    stamp = maybe_stamp(ctx)
-
```

Comment:
This stuff was just moved to a separate file for spm_publishing. Just an organizational change.